Quadriga/src/main/java/edu/asu/spring/quadriga/aspects/ConceptCollectionAuthorization.java

```diff
@@ -30,28 +33,27 @@
     @Autowired
     private IConceptCollectionFactory collectionFactory;
 
-    /**
-     * This method checks the access permissions for the given ConceptCollection
-     * for the logged in user.
-     * 
-     * @param userName
-     *            - logged in user name.
-     * @param conceptCollectionId
-     *            - Concept Collection id.
-     * @param userRoles
-     *            - the roles for which the user should be checked against the
-     *            Concept Collection.
-     * @throws QuadrigaAccessException
-     *             ,QuadrigaStorageException
-     * @author Kiran Batna
-     */
+    private final Logger logger = LoggerFactory.getLogger(ConceptCollectionAuthorization.class);
+
     @Override
-    public boolean chkAuthorization(String userName,
-            String conceptCollectionId, String[] userRoles)
+    public boolean chkAuthorization(String userName, Object conceptCollectionObj, String[] userRoles)
             throws QuadrigaStorageException, QuadrigaAccessException {
-        
+
+        IConceptCollection collection;
+        String conceptCollectionId = null;
```

Comment:
Instead of having two variables, I'd just use a collection variable and call its getId method when you need the id. It'll make your code more compact and readable.